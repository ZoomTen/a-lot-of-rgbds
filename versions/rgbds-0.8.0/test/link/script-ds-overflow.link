rom0
	ds $4000 ; OK
	ds 1 ; Boom!

hram
	ds $7F ; OK
	ds $FFFF ; Boom!
