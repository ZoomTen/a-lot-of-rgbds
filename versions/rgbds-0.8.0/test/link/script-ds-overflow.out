error: script-ds-overflow.link(3): Cannot increase the current address by 1 bytes: only 0 bytes to $4000
error: script-ds-overflow.link(7): Cannot increase the current address by 65535 bytes: only 0 bytes to $ffff
Linking failed with 2 errors
